bin/webpack-configs.js

```diff
@@ -785,11 +785,17 @@ const getInteractivityAPIConfig = ( options = {} ) => {
 	const { alias, resolvePlugins = [] } = options;
 	return {
 		entry: {
-			runtime: './assets/js/interactivity',
+			'wc-interactivity': './assets/js/interactivity',
 		},
 		output: {
-			filename: 'woo-directives-[name].js',
```

Comment:
Is there a specific reason for this change?

Reply:
It's just that I changed the name to make it consistent with the format used in other WooCommerce scripts in the repository. Nothing important. 😄

Note that there's still a WooCommerce prefix in that file; the name of the output file has turned from `woo-directives-runtime.js` to `wc-interactivity.js`.

Reply:
I also removed the `vendors` chunk, so I saw it unnecessary to differentiate between `runtime` and `vendors`.